### ISSUE TITLE
TLSA records don't match cert chain for "jpbe.de" MX-hosted domains

### ISSUE THREAD

**Issue by vdukhovni** [danefail/list#39]
```
jpbe.de
6in4.de
as206479.net
inboxcheck.de
jnpl.de
jpbe-apps.de
```

Details:
```
jpbe.de. IN MX 10 homer.jpbe.de.
jpbe.de. IN MX 100 milhouse.jpbe.de.
  homer.jpbe.de[185.120.22.2]: tlsa-mismatch
  homer.jpbe.de[2001:67c:16c8:2::1]: tlsa-mismatch
  milhouse.jpbe.de[185.120.22.3]: tlsa-mismatch

inboxcheck.de. IN MX 5 homer.jpbe.de.
jpbe-apps.de. IN MX 10 homer.jpbe.de.
as206479.net. IN MX 10 homer.jpbe.de.
jnpl.de. IN MX 10 homer.jpbe.de.
6in4.de. IN MX 50 homer.jpbe.de.
```

**Comment by desh-se** [comment]
I've contacted the domain owner using the web form on jpbe.de.

**Comment by desh-se** [comment]
Closed by d283ae0aef1ac646c1da170669b79913d399a42a
